File: back-end/src/main/java/com/nvm/project1/service/ColorService.java
package com.nvm.project1.service;

import com.nvm.project1.entity.Color;
import com.nvm.project1.request.ColorRequest;
import com.nvm.project1.response.ColorResponse;

import java.util.List;

public interface ColorService {

    List<ColorResponse> getAll();


    ColorResponse add(ColorRequest colorRequest);

}

File: font-end/src/components/services/colorService.js
import axios from "axios";


class ColorService{
    getAll=()=>{
        return axios.get(`http://localhost:8080/color/list`)
    }

    addColor=(colorRequest)=>{
        return axios.post(`http://localhost:8080/color/add`,colorRequest)
    }
}

export default new ColorService();